plaso/cli/status_view.py

```diff
@@ -389,14 +393,18 @@ def SetMode(self, mode):
     """
     self._mode = mode
 
-  def SetSourceInformation(self, source_path, source_type, filter_file=None):
+  def SetSourceInformation(
+      self, source_path, source_type, artifact_filters=None,
```

Comment:
lines 397 and 398 look like they can fit on a single line

Reply:
Done